### PR TITLE
Additional AD properties

### PR DESCRIPTION
**Pull request by JonasBK**
## Description

Add additional AD properties and pre-defined searches

## Motivation and Context

This PR addresses: BP-792

Enables more to find more non-path security issues.

Should be merged and included in a release simultaneously or after this SharpHound change: https://github.com/BloodHoundAD/SharpHoundCommon/pull/150

## How Has This Been Tested?

Locally.

## Screenshots (optional):

image

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have met the contributing prerequisites
  - Assigned myself to this PR
  - Added the appropriate labels
  - Associated an issue: https://github.com/SpecterOps/BloodHound/issues/672
  - Read the Contributing guide: https://github.com/SpecterOps/BloodHound/wiki/Contributing
- [x] I have ensured that related documentation is up-to-date
  - Open API docs
  - Code comments (GoDocs / JSDocs)
- [x] I have followed proper test practices
  - Added/updated tests to cover my changes
  - All new and existing tests passed

### CODE REVIEW

File: packages/javascript/bh-shared-ui/src/commonSearches.tsx [hunk 1]
```diff
@@ -210,6 +210,34 @@ export const CommonSearches: CommonSearchType[] = [
                 description: 'Enabled Tier Zero / High Value principals inactive for 60 days',
                 cypher: `WITH 60 as inactive_days\nMATCH (n)\nWHERE n.system_tags CONTAINS "admin_tier_0"\nAND n.enabled = true\nAND n.lastlogontimestamp < (datetime().epochseconds - (inactive_days * 86400)) // Replicated value\nAND n.lastlogon < (datetime().epochseconds - (inactive_days * 86400)) // Non-replicated value\nAND n.whencreated < (datetime().epochseconds - (inactive_days * 86400)) // Exclude recently created principals\nAND NOT n.name STARTS WITH "AZUREADKERBEROS." // Removes false positive, Azure KRBTGT\nAND NOT n.objectid ENDS WITH "-500" // Removes false positive, built-in Administrator\nAND NOT n.name STARTS WITH "AZUREADSSOACC." // Removes false positive, Entra Seamless SSO\nRETURN n`,
             },
+            {
```
Comment (urangel) [inline packages/javascript/bh-shared-ui/src/commonSearches.tsx:213]: There was an update to set limits on all of these prebuilt queries recently. Would it be good to put limits on these new queries as well?

**Review by urangel — APPROVED** [review]
Consider adding limits to the new prebuilt queries similar to the recent updates from https://github.com/SpecterOps/BloodHound/pull/773 but if they are not necessary the changes look good 👍

File: packages/javascript/bh-shared-ui/src/commonSearches.tsx (hunk 1 above)
Comment (JonasBK) [reply packages/javascript/bh-shared-ui/src/commonSearches.tsx:213]: Good call, will update 👍

**Comment by AkechiShiro** [comment]
Hello @JonasBK  thanks for improvements, I'm curious as to which SharpHound version/release will we have to use to benefit from : https://github.com/BloodHoundAD/SharpHoundCommon/pull/150 since this looks like a library for SharpHound, I'm not sure if the v4 is used at the moment for the latest release of SharpHound.

**Comment by StephenHinck** [comment]
> Hello @JonasBK thanks for improvements, I'm curious as to which SharpHound version/release will we have to use to benefit from : [BloodHoundAD/SharpHoundCommon#150](https://github.com/BloodHoundAD/SharpHoundCommon/pull/150) since this looks like a library for SharpHound, I'm not sure if the v4 is used at the moment for the latest release of SharpHound.
> 
> I see that the latest release is `v4.0.4` beta from 2 weeks ago, I take it that this additional AD properties will come in the next release that will be made for the library but for upstream (SharpHound collector), we're still using v3 which is stable.

This should be included, not in our next release, but the one following of BloodHound / SharpHound, tentatively scheduled for the week of September 9th.

**Pull request merged**